camera-processor/src/CameraProcessor.ts

```diff
@@ -59,8 +55,12 @@ class CameraProcessor<TAnalyzerData = any> {
     this.performance.frameTime.analyze = time_analyze - time_start;
     this.performance.frameTime.render = time_render - time_analyze;
     this.performance.frameTime.total = time_render - time_start;
-    this.performance.fps = Math.min(1000 / this.performance.frameTime.total, 1000);
-    if (this.isRunning) requestAnimationFrame(this.processFrame);
+
+    // Time precision of Date.now() is not up to a millisecond for security reasons, so we may end up having zero frame time.
+    // More info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now#reduced_time_precision
+    this.performance.fps = this.performance.frameTime.total > 0 ? this.performance.frameTime.total * 0.001 : 1000;
```

Comment:
1000 / x != x * 0.001
Should be: this.performance.frameTime.total > 0 ? 1000 / this.performance.frameTime.total : 1000;

Reply:
Yes, my bad, I have turned the expression upside down. Anyway, using the ternary operator should yield faster performance
```suggestion
    this.performance.fps = this.performance.frameTime.total > 0 ? 1000 / this.performance.frameTime.total : 1000;
```